apiserver/api.go

```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/rs/cors"
 
-	"github.com/fiorix/freegeoip"
+	"github.com/ashleym1972/freegeoip"
```

Comment:
This needs to be fixed otherwise `go get` will fail. Usually when you clone go-based projects you keep the original absolute path for imports and just add a different git remote entry to it.